WooCommerce/Classes/ViewRelated/Orders/Order Creation/Taxes/NewTaxRateSelectorView.swift

```diff
@@ -37,65 +37,46 @@ struct NewTaxRateSelectorView: View {
+                            .background(Color(.listForeground(modal: false)))
 
-                                Image(systemName: "chevron.right")
-                                    .font(.body)
-                                    .font(Font.title.weight(.semibold))
-                                    .foregroundColor(Color(.textTertiary))
-                                    .padding(.leading, Layout.generalPadding)
+                            bottomNotice
+                            .renderedIf(!viewModel.shouldShowBottomActivityIndicator)
+                        }
+                    case .empty:
+                        EmptyState(title: "",
+                                   description: "",
+                                   image: .emptyInboxNotesImage)
+                        .frame(maxHeight: .infinity)
+                    case .syncingFirstPage:
+                        ScrollView {
+                            LazyVStack(spacing: 0) {
+                                ForEach(viewModel.placeholderRowViewModels, id: \.id) { rowViewModel in
+                                    TaxRateRow(viewModel: rowViewModel)
+                                        .redacted(reason: .placeholder)
+                                        .shimmering()
+                                }
                             }
-                            .padding(Layout.generalPadding)
                         }
-                    }
-
-                    Divider()
-                }
-
-                Text(Localization.editTaxRatesInWpAdminSectionTitle)
-                    .foregroundColor(Color(.textSubtle))
-                    .footnoteStyle()
-                    .padding(.top, Layout.editTaxRatesInWpAdminSectionTopPadding)
-                    .frame(maxWidth: .infinity, alignment: .center)
-                    .padding([.leading, .trailing], Layout.generalPadding)
-
-                Button(action: {
-                    showingWPAdminWebview = true
-                }) {
-                    HStack {
-                        Text(Localization.editTaxRatesInWpAdminButtonTitle)
-                            .fontWeight(.semibold)
-                            .font(.footnote)
-                            .foregroundColor(Color(.wooCommercePurple(.shade60)))
-
-                        Image(systemName: "arrow.up.forward.square")
-                    }
-                    .frame(maxWidth: .infinity, alignment: .center)
+                        .background(Color(.listForeground(modal: false)))
                 }
-                .padding(.top, Layout.editTaxRatesInWpAdminSectionVerticalSpacing)
-                .safariSheet(isPresented: $showingWPAdminWebview, url: viewModel.wpAdminTaxSettingsURL, onDismiss: {
-                    onDismissWpAdminWebView()
-                    showingWPAdminWebview = false
-                })
-
-                Spacer()
-
+            }
+            .onAppear {
+                viewModel.onLoadTrigger.send()
```

Comment:
`onLoad` and `onAppear` have different semantic implications to me when reading the code. I expect `onLoad` to be called once in the lifecycle, and `onAppear` to be callable multiple times. Perhaps this is out of date in SwiftUI; I can't really remember which this will be.

If it'll happen more than once, I'd suggest renaming the view model's property.